howto/install/deploy-juju.md

```diff
@@ -10,7 +10,7 @@ There are differences between the full Anbox Cloud installation and the Anbox Cl
 
 Before you start the installation, ensure that you have the required credentials and prerequisites:
 
-* An Ubuntu 18.04 LTS or 20.04 LTS environment to run the commands (or another operating system that supports snaps - see the [Snapcraft documentation](https://snapcraft.io/docs/installing-snapd)).
+* An Ubuntu 18.04 LTS, 20.04 LTS, or 22.04 LTS environment to run the commands (or another operating system that supports snaps - see the [Snapcraft documentation](https://snapcraft.io/docs/installing-snapd)).
```

Comment:
Ah, good point: We have to drop Ubuntu 18.04 here as we no longer support it starting with 1.16

I missed on that, can you add that here or should I provide a separate PR?

Reply:
Can we also declare 22.04 as the recommended version to use?

Reply:
Updated.

But we still support the bionic images, correct?
https://anbox-cloud.io/docs/ref/provided-images